linters/README.md

```diff
@@ -5,3 +5,5 @@ Files to configure the linters we use for each language and technology
 
 * Ruby
   * [Rubocop](/linters/ruby/.rubocop.yml)
+* JavaScript
```

Comment:
Rubocop does not enforce a version of Ruby nor of any of it's libraries, I don't see how that applies.

But yes, if we all agree to use ES6 and use Babel as the transpiler, that should be written elsewhere. What about having ES6 linter, instead of JS linter? We don't do that for Ruby because the linter does not care about versions, but if in JS it does, maybe we should have the version specified.

Also, we can add to the guides that you should prefer ES6, but that might not always be possible, for instance in projects we don't start or have technological impositions by the client.